_release-pypi/action.yml

```diff
@@ -22,6 +22,12 @@ inputs:
     description: 'Desired Python version for using twine'
     required: true
     type: string
+  download-artifacts:
```

Comment:
For "private" actions (i.e. starting with "_"), it is better not to have optional args.

Reply:
Fair enough